nixos/modules/services/web-apps/dolibarr/default.nix

```diff
@@ -0,0 +1,237 @@
+{ config, lib, pkgs, options, ... }:
+let
+  cfg = config.services.dolibarr;
+  app = "dolibarr";
```

Comment:
What is the intention to use this variable in a lot of places? It makes the code less readable.